basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository.component/src/main/resources/application.properties

```diff
@@ -12,4 +12,4 @@ basyx.backend = InMemory
 #spring.data.mongodb.password=password
 
 # Base Path for Spring Boot Actuator
-management.endpoints.web.base-path=/
+management.endpoints.web.base-path=/
```

Comment:
Please revert this file to its original state

Reply:
Updated in the latest commit